pkg/daemon/ceph/client/image_test.go

```diff
@@ -29,9 +29,18 @@ import (
 )
 
 const (
-	sizeMB = 1048576 // 1 MB
+	sizeMB = MiB // 1 MB
 )
 
+func TestRoundup_size_MiB(t *testing.T) {
+
```

Comment:
Please remove blank lines at the start of a code block. It's generally preferred, though not strictly enforced

Reply:
gone